icicle-compiler/src/Icicle/Sea/IO/Psv/Output.hs

```diff
@@ -411,6 +415,10 @@ seaOfOutput isJSON struct structIndex outId missing env outType argTypes transfo
    seaOfOutputBase' b
      = seaOfOutputBase b mismatch
 
+seaOfNotAnError :: Doc -> Doc
+seaOfNotAnError v =
+  "((" <> v <> " == ierror_not_an_error) || (" <> v <> " == ierror_tombstone))"
```

Comment:
confused. maybe this should just be true. shouldn't we print NA for all errors (cannot compute, fold1 no value, tombstone, etc)?